src/components/package/FileExplorer.vue

```diff
@@ -58,7 +58,7 @@
         />
         {{
           filesLocation === "" || filesLocation === "/"
-            ? "Return to package review"
+            ? "Return to module review"
```

Comment:
While we're at it, something like 
```suggestion
            ? "Return to module overview"
```
would feel a bit more natural to me as a user, but it should be discussed whether there are any better alternatives.

Reply:
Agreed!